src/DatePicker.jsx

```diff
@@ -0,0 +1,26 @@
+import { DateRangePicker } from '@mui/x-date-pickers-pro';
+import { LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { useState } from "react";
+
+const CustomDatePicker = ({ selectedDateRange, onRangeChange }) => {
```

Comment:
Where are the params being passed?

Reply:
I've also updated App.jsx file. Could you please try again?